lib/block-supports/duotone.php

```diff
@@ -311,9 +311,6 @@ function gutenberg_get_duotone_filter_id( $preset ) {
  * @return string        Duotone CSS filter property url value.
  */
 function gutenberg_get_duotone_filter_property( $preset ) {
-	if ( isset( $preset['colors'] ) && 'unset' === $preset['colors'] ) {
-		return 'none';
```

Comment:
I wish I would have used `'none'` as the string instead of `'unset'` so that string values could just be treated as CSS like the rest of the block supports. But for maintaining backwards compatibility, I don't think that's really an option anymore.

Reply:
Just so you know this return value is preserved but I moved it outside of this function so it doesn't need `colors` in the "preset".